app/frontend/src/components/designer/FormViewerMultiUpload.vue

```diff
@@ -163,26 +166,39 @@ export default {
       progress: false,
       report_file_name: undefined,
       max_file_size: 5,
-      ERROR: {
-        UPLOAD_MULTIPLE_FILE_ERROR: 'Sorry, you can upload only one file',
-        DRAG_MULPLE_FILE_ERROR: 'Sorry, you can drag only one file',
-        FILE_FORMAT_ERROR: 'Sorry, we only accept json files',
-        FILE_SIZE_ERROR: 'Max file size allowed is 5MB',
-        PARSE_JSON_ERROR: 'We can not parse json data from the file',
-        JSON_OBJECT_NOT_ARRAY: 'Wrong json file format',
-        JSON_ARRAY_EMPTY: 'This json file is empty.',
-        ERROR_WHILE_VALIDATE: 'There is something wrong with this file',
-        ERROR_WHILE_CHECKVALIDITY: 'There is something wrong with this file',
-        ERROR_AFTER_VALIDATE:
-          'Some errors found, see below for more information.',
-      },
     };
   },
   computed: {
     txt_color() {
       if (!this.error) return 'success-text';
       else return 'fail-text';
     },
+    ERROR() {
```

Comment:
Why is a computed value necessary? None of these values are actually being computed, shouldn't you just directly use the translations where they are needed?

Reply:
Because the health team declared the variables. I just did the translations

Reply:
Because the health team declared the variables. I just did the translations

Reply:
We should remove this computed value and use the direct translations where they are needed.